index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <script  src="js/index.js"></script>
+    <h1 id="title">Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+    <form id="survey-form">
+      <input id="name" type="text" value="Badru Rohit" placeholder="input name"><label id="name-label"></label>Name</input><br>
+      <input id="email" type="email" value="
+rohitmurari@gmail.com"placeholder="input email"><label id="email-label"></label>Email</input><br>
+<input id="number"  min="5" max="50" placeholder="input" ><label id="number-label"></label> age</input><br>
+<select id="dropdown"> <option value="volvo">1</option>
+  <option value="2">2</option>
+  <option value="3">3</option>
+  <option value="4">4</option></select><br>
+  <div>Are you happy?</div>
+  <input type="radio" id="yes" name="fav_language" >yes<br>
+<label for="yes"></label>
+<input type="radio" id="no" name="fav_language" >
+<label for="no">No</label><br>
+
+<div>What would you like to see improved?</div>
+<input type="checkbox" id="project1" name="project1" value="Front">
+<label for="project1"> Front end projects</label><br>
+<input type="checkbox" id="project2" name="project2" value="Back">
+<label for="project2"> Back end projects</label><br>
+<input type="checkbox" id="project3" name="project3" value="Data">
+<label for="project3"> Data Visualisation</label><br>
+<br>
+<label for="comments">Any comments or suggestions?</label>
+
+<textarea id="comments" name="suggestions" rows="4" cols="50">
+
+
+</textarea>
+
+<input type="submit" value="Submit" >
+
+    </form>
```

Comment:
Please try to break up this section and wrap it in semantic tags and html tags to structure it better.
please assign classes to the sections of the code and position ,style and adjust it to your preference